server/aws-lsp-codewhisperer/src/language-server/telemetry/types.ts

```diff
@@ -102,23 +102,110 @@ export enum ChatTelemetryEventName {
     ExitFocusChat = 'ExitFocusChat',
     EnterFocusConversation = 'EnterFocusConversation',
     ExitFocusConversation = 'ExitFocusConversation',
+    StartConversation = 'StartConversation',
+    InteractWithMessage = 'InteractWithMessage',
+    AddMessage = 'AddMessage',
+    RunCommand = 'RunCommand',
+    MessageResponseError = 'MessageResponseError',
```

Comment:
These names are incorrect, I think they should be `amazonq_....`. I will update that in https://github.com/aws/language-servers/pull/330